develop/index.md

```diff
@@ -6,29 +6,193 @@ hide:
+    - [**Introduction to XCM**](/develop/interoperability/intro-to-xcm/){target=\_blank} - introduces key concepts, core function definitions, and code examples
+    - [**XCM Tools**](/develop/toolkit/interoperability/xcm-tools/){target=\_blank} - provides an overview of popular XCM tools
+    - [**Tutorials for Managing XCM Channels**](/tutorials/interoperability/xcm-channels/){target=\_blank} - guides for using [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank} UI to establish cross-chain messaging channels 
 
-- **Cross-chain developers** - harness Polkadot's interoperability features to enable secure communication and asset transfers across blockchains. Leverage Cross-Consensus Messaging (XCM) to create innovative cross-chain workflows and applications
 
-    - **Where to start** - [Introduction to XCM](/develop/interoperability/intro-to-xcm/){target=\_blank}
+### Parachain Developers 
+
+Build, deploy, and maintain custom parachains with the Polkadot SDK.
+
+####  Learn
+
+<div class="grid cards" markdown>
+
+- :octicons-book-16:{ .lg .middle } [Parachains Overview](/polkadot-protocol/architecture/parachains/overview/){target=\_blank}
+
+- :octicons-book-16:{ .lg .middle } [Custom Parachains](/develop/parachains/get-started/build-custom-parachains/){target=\_blank}
+
+- :octicons-book-16:{ .lg .middle } [Introduction to the Polkadot SDK](/develop/parachains/get-started/intro-polkadot-sdk/){target=\_blank}
+
+</div>
+
+#### Build
+
+<div class="grid cards" markdown>
+
+
+- :octicons-rocket-16:{.lg .middle} [Build and Deploy Your First Parachain](TODO: update-path){target=\_blank}
+
+- :octicons-rocket-16:{.lg .middle} [Install Polkadot SDK Dependencies](/develop/parachains/get-started/install-polkadot-sdk/){target=\_blank}
+
+- :octicons-rocket-16:{.lg .middle} [Spawn a Basic Chain for Testing with Zombienet](/tutorials/polkadot-sdk/testing/spawn-basic-chain/){target=\_blank}
+
+- :octicons-rocket-16:{.lg .middle} [Fork a Chain for Testing with Chopsticks](/tutorials/polkadot-sdk/testing/fork-live-chains/){target=\_blank}
+
+</div>
+
+#### Tools
+
+<div class="grid cards" markdown>
+
+- :octicons-tools-16:{.lg .middle} [Parachain Template](https://github.com/paritytech/polkadot-sdk/tree/master/templates/parachain){target=\_blank}
+
+- :octicons-tools-16:{.lg .middle} [Open Zeppelin Polkadot Runtime Templates](https://github.com/OpenZeppelin/polkadot-runtime-templates/tree/main){target=\_blank}
+
+</div>
+
+#### Resources
+
+<div class="grid cards" markdown>
+
+- :octicons-code-square-16:{.lg .middle} [Polkadot SDK source code repository](https://github.com/paritytech/polkadot-sdk){target=\_blank} 
+    
+- :octicons-code-square-16:{.lg .middle} [FRAME source code repository](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame){target=\_blank}
+
+</div>
+
+### Smart Contract Developers 
+
+Leverage smart contracts and execute custom logic over existing chains to streamline your development process.
```

Comment:
Please link to this: https://forum.polkadot.network/t/contracts-on-assethub-roadmap/9513/57

Reply:
Also, I see that @CrackTheCode016 the has this great thread about the state of things. https://x.com/baderyo_o/status/1864081312090341879

Should have a small page that explains the transition to smart contracts, and link to anywhere that we have this banner?